css/css-text/letter-spacing/reference/letter-spacing-cursive-001-ref.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<head>
+<meta charset="utf-8">
+<title>
+CSS Text Test - Letter-spacing should not be applied for Cursive Scripts.
+</title>
+<link rel="author" title="Sejal Anand" href="mailto:sejalanand@microsoft.com">
+<style>
+div {
+  margin: 1em;
+  font: 24px serif;
+  white-space: nowrap;
+  display: inline-block;
+  outline: 1px solid gray;
+}
+</style>
+</head>
+<body>
+<p>
+The following test will pass if no spacing appears between letters within
+Arabic words. Both Arabic texts should display identically.
+</p>
+<div>
+<span>١. مرحباً</span>
+</div>
+<br>
+<div>
+<span>٢. مرحباً</span>
```

Comment:
If we want both pieces of text to render identically (regardless of letter spacing), they should actually be the same text.

```suggestion
<span>١. مرحباً</span>
```

Reply:
done